src/select2.js

```diff
@@ -244,40 +61,29 @@ angular.module("rt.select2", [])
                     select2Stack.$unregister(controlObj);
                 });
 
+                // initiate select2
                 $timeout(function () {
                     element.select2(opts);
-                    element.on("change", function (e) {
-                        scope.$apply(function () {
-                            var val;
-                            if (isMultiple) {
-                                var vals = [];
-                                for (var i = 0; i < e.val.length; i++) {
-                                    val = optionItems[e.val[i]];
-                                    if (val) {
-                                        vals.push(val.id);
-                                    }
-                                }
-                                controller.$setViewValue(vals);
-                            } else {
-                                val = optionItems[e.val];
-                                controller.$setViewValue(val ? val.id : null);
-                            }
-
-                            controller.$render();
-
-                        });
-                    });
 
                     element.on("select2-blur", function () {
```

Comment:
I might as well mention that there is no longer a `select2-blur` event that is triggered.